tests/testsuite/publish.rs

```diff
@@ -192,12 +179,14 @@ See [..]
         )
         .run();
 
-    validate_upload_foo();
+    // Skip `validate_upload_foo` as we just cared we got far enough for verify the token behavior.
+    // Other tests will verify the endpoint gets the right payload.
```

Comment:
Sorry I did't get it. Could you guide me through the place that verifies this?

Reply:
Context: this is the `old_token_location` test

This test is focused on authentication.  Once we have been authenticated, the token location should not make a difference to formatting of the output we give to the payload.  We verify the publish succeeds using `with_stderr` and seeing the `[UPLOADING]` without any errors after it.  At that point, we know the old location works.  Other tests, like `simple` focus on verifying we give the right data to the end point.